src/pages/UserSettingsPage/UserSettingsPage.tsx

```diff
@@ -0,0 +1,285 @@
+import { Box, Button, TextField, Select, InputLabel, MenuItem, FormControl } from '@mui/material';
+import { Controller, useForm } from 'react-hook-form';
+
+interface UserInputs {
+  username: string;
+  email: string;
+  name: string;
+  age: number;
+  gender: string;
+  user_pic: string;
+  user_bio: string;
+  max_distance: number;
+  zipcode: number;
+  // creation_time: string;
+  // last_update: string | null;
+}
+
+const UserSettingsPage: React.FC = (): React.ReactElement => {
+  const { control, handleSubmit } = useForm<UserInputs>({
+    mode: 'onBlur',
+    reValidateMode: 'onBlur',
+    defaultValues: {
+      username: '',
+      email: '',
+      name: '',
+      age: 0,
+      gender: '',
+      user_pic: '',
+      user_bio: '',
+      max_distance: 0,
+      zipcode: 88901,
+    }
+  });
+
+  return (
+    <Box
+      sx={{
+        mt: 2,
+        display: 'flex',
+        flexDirection: 'column',
+        justifyContent: 'center',
+        alignItems: 'center',
+      }}
+    >
+      <Box
+        sx={{
+          width: '25%',
+          maxWidth: '350px',
+          minWidth: '250px' }}
+      >
+        <form onSubmit={handleSubmit((values) => console.log(values))}>
+          <Controller
+            control={control}
+            name="username"
+            render={({
+              field: { ref, onChange, onBlur, value, name },
+              fieldState: { isTouched, isDirty, error },
+              formState,
+            }) => (
+              <TextField
+                label="Username"
+                // TODO: Can users change their usernames?
```

Comment:
They can, but username as a whole will probably be removed (since what we're really using for login is email)

Reply:
That being the case, I'll leave this in for now. When we have the username in the token and users can log in fully, We can just display the username on the page.